dev/com.ibm.ws.persistence.ol_fat/test-applications/consumer/src/com/ibm/ws/persistence/consumer/web/ConsumerServlet.java

```diff
@@ -0,0 +1,330 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.ws.persistence.consumer.web;
+
+import java.io.StringWriter;
+import java.util.Arrays;
+
+import javax.annotation.Resource;
+import javax.ejb.EJB;
+import javax.servlet.annotation.WebServlet;
+import javax.transaction.RollbackException;
+import javax.transaction.UserTransaction;
+
+import com.ibm.websphere.persistence.test.Consumer;
+import com.ibm.websphere.persistence.test.Consumer.DATABASE;
+import com.ibm.ws.persistence.consumer.ejb.PersonBean;
+import com.ibm.ws.persistence.consumer.model.Person;
+
+@WebServlet("/*")
+@SuppressWarnings("serial")
+public class ConsumerServlet extends AbstractServlet {
```

Comment:
per earlier comment, swap to use FATServlet

Reply:
Done